src/mame/layout/tvochken.lay

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="card">
+		<!-- force maximum state -->
+		<rect state="0"><color alpha="0" /></rect>
+		<rect state="49"><color alpha="0" /></rect>
+	</element>
+
+	<view name="Default View">
+		<screen index="0">
+			<bounds x="0" y="0" width="4" height="3" />
+		</screen>
+
+		<element id="card" ref="card">
+			<bounds x="0" y="3.1" width="4" height="2.66666" />
```

Comment:
Actual ratio is 3/2, is there a more accurate way of writing this here?

Reply:
Looks fine to me? You could add a comment if you want
```<!-- card ratio is 3/2 -->```